server/model/monitor.js

```diff
@@ -347,6 +350,33 @@ class Monitor extends BeanModel {
                         throw new Error("Server not found on Steam");
                     }
 
+                } else if (this.type === "docker") {
+                    debug(`[${this.name}] Prepare Options for Axios`);
+
+                    const options = {
+                        url: `/containers/${this.docker_container}/json`,
+                        headers: {
+                            "Accept": "*/*",
+                            "User-Agent": "Uptime-Kuma/" + version,
+                        },
+                        httpsAgent: new https.Agent({
+                            maxCachedSessions: 0,      // Use Custom agent to disable session reuse (https://github.com/nodejs/node/issues/3940)
+                            rejectUnauthorized: ! this.getIgnoreTls(),
+                        }),
+                    };
+
+                    if (this.docker_type === "socket") {
+                        options.socketPath = this.docker_daemon;
+                    } else if (this.docker_type === "tcp") {
+                        options.baseURL = this.docker_daemon;
+                    }
+
+                    debug(`[${this.name}] Axios Request`);
+                    let res = await axios.request(options);
+                    if (res.data.State.Running) {
+                        bean.status = UP;
+                        bean.msg = "";
+                    }
```

Comment:
```suggestion
                    }
                    
```

Perhaps add a new line here to match up with before and the rest of the `if else` block.